src/Link_TEST.cc

```diff
@@ -763,6 +763,50 @@ TEST(DOMLink, InertialValuesGivenWithAutoSetToTrue)
   root.ResolveAutoInertials(errors, sdfParserConfig);
   EXPECT_TRUE(errors.empty());
 
+  EXPECT_DOUBLE_EQ(4.0, link->Inertial().MassMatrix().Mass());
+  EXPECT_EQ(gz::math::Pose3d::Zero, link->Inertial().Pose());
+  EXPECT_EQ(gz::math::Vector3d(0.66666, 0.66666, 0.66666),
+    link->Inertial().MassMatrix().DiagonalMoments());
+}
+
+/////////////////////////////////////////////////
+TEST(DOMLink, InertialValuesGivenWithAutoSetToTrue)
+{
+  std::string sdf = "<?xml version=\"1.0\"?>"
+  "<sdf version=\"1.11\">"
+  "  <model name='compound_model'>"
+  "   <link name='compound_link'>"
+  "     <inertial auto='true'>"
+  "       <pose>1 1 1 2 2 2</pose>"
+  "       <inertia>"
```

Comment:
the diff is a little confusing here, but I think a new test `ResolveAutoInertialsWithMass` has been added above the existing `InertialValuesGivenWithAutoSetToTrue` test, and that the existing test has been modified, with the `<mass>` removed from the line before this one

if you move the new test between `ResolveAutoInertialsCalledWithAutoFalse` and `AddCollision`, it may clarify the diff

can you confirm and either revert the change to the existing test if it's not needed or add a comment about why it is changed?

Reply:
moved test so the diff is not easier to read. I duplicated the test so that there's now one test with mass specified and one without mass. [cf4d77b](https://github.com/gazebosim/sdformat/pull/1513/commits/cf4d77b136deb3a28b757686a494de3b087fa4eb)

Reply:
thanks! There was one expectation in this test that I thought should be updated as well, but it was outside the lines that I could add a comment on, so I just changed it directly in https://github.com/gazebosim/sdformat/pull/1513/commits/ae7685c34c85a751c9da405ec321ad67f7e27232